examples/models/rex_quadrotor.jl

```diff
@@ -0,0 +1,160 @@
+using Rotations
+using ForwardDiff
+using LinearAlgebra
+# using ControlSystems
+using JSON
+using StaticArrays
+using Printf
+using RobotDynamics: ContinuousDynamics, RigidBody, LieGroupModel, @autodiff
+import RobotDynamics: dynamics!, dynamics, jacobian!, forces, moments, wrenches, inertia, inertia_inv, orientation
+import RobotDynamics: state_dim, control_dim
+
+const quad_mass = 2.0
+const quad_inertia = SMatrix{3, 3}([0.01566089 0.00000318037 0; 0.00000318037 0.01562078 0; 0 0 0.02226868])
+const quad_motor_kf = 0.0244101
+const quad_motor_bf = -30.48576
+const quad_motor_km = 0.00029958
+const quad_motor_bm = -0.367697
+const quad_arm_len = 0.28
+const quad_min_throttle = 1148.0
+const quad_max_throttle = 1832.0
+
+@autodiff struct RexQuadrotor <: RigidBody
+    mass::Float64
+    J::SMatrix{3,3,Float64,9}
+    Jinv::SMatrix{3,3,Float64,9}
+    gravity::SVector{3,Float64}
+    motor_dist::Float64
+    kf::Float64
+    km::Float64
+    bf::Float64
+    bm::Float64
+end
+
+control_dim(::RexQuadrotor) = 4
```

Comment:
I prefer to avoid using `import` and instead explicitly specify where I'm extending methods from another module.